### PR TITLE
Improved config ui

### LINKED ISSUE #81: Improve the configuration UI
Share styles between different notebooks with UI.
Give the UI a more professional look.
Validate the password to the secret store when it's entered.
Pop up an error message if the password is invalid.

### PR DESCRIPTION
**Pull request by ahsimb**
closes #81

### CODE REVIEW

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/access_store_ui.ipynb [hunk 1]
```diff
@@ -10,6 +10,31 @@
...
+   "execution_count": null,
+   "id": "74f55112-92dc-4a54-ac1a-b068fadfbcc7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "\n",
+    "def upward_file_search(file_name: str) -> str:\n",
+    "\n",
+    "    dir = Path().resolve()\n",
```
Comment (ckunki) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/access_store_ui.ipynb:24]: Could we move this functionality to notebook-connector to enable to apply pytest test cases?

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/access_store_ui.ipynb [hunk 2]
```diff
@@ -47,25 +59,48 @@
...
+    "        display(Javascript(f\"alert('{message}')\"))\n",
     "\n",
     "    def open_or_create_config_store(btn):\n",
     "        global sb_config, sb_store_file\n",
-    "        sb_store_file = inputs[WKey.file].value\n",
-    "        sb_config = Secrets(Path(root_dir) / sb_store_file, inputs[WKey.password].value)\n",
-    "        # Save the file in the shared store.\n",
-    "        %store sb_store_file\n",
-    "        del sb_store_file\n",
+    "        sb_store_file = file_txt.value\n",
```
Comment (ckunki) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/access_store_ui.ipynb:82]: If you prefer, then we could also add this kind of functionality as convenience to the secret store....

**Review by ckunki — CHANGES_REQUESTED** [review]

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/access_store_ui.ipynb (hunk 2 above)
Comment (ahsimb) [reply exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/access_store_ui.ipynb:82]: I would leave it in a notebook, to be honest. It's not impossible to call the ipython implementation of "line magic", but I think you need a good reason for that.

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/access_store_ui.ipynb (hunk 1 above)
Comment (ahsimb) [reply exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/access_store_ui.ipynb:24]: Yes, it's a good idea.

**Pull request merged**